### PR TITLE
Expose daml3-script to sdk

### PR DESCRIPTION
**Pull request by samuel-williams-da**
Adds `daml3-script` as a dependency until daml3 releases, in which it'll replace daml-script directly.
Requires LF version of 1.dev, and errors otherwise to use.

### CODE REVIEW

**Review by garyverhaegen-da — APPROVED** [review]
LGTM, thanks!

**Comment by garyverhaegen-da** [comment]
Is the addition of the daml-ghcide submodule deliberate? If so, I think it warrants some extra docs around it.

**Comment by samuel-williams-da** [comment]
Oh good catch! I'm not sure why my stash didnt grab that

**Comment by remyhaemmerle-da** [comment]
Could you add a HUGE warning at daml-script-3 startup, saying this is an unsuported feature ?

File: compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs [hunk 1]
```diff
@@ -553,7 +554,11 @@ expandSdkPackages lfVersion dars = do
       | fp `elem` basePackages = pure fp
       | isSdkPackage fp = case mbSdkPath of
             Just _ | isInvalidDaml3Script fp -> fail "Daml3-script may only be used in LF 1.dev, and is unstable."
-            Just sdkPath -> pure $ sdkPath </> "daml-libs" </> fp <> sdkSuffix <.> "dar"
+            Just sdkPath -> do
+              when (fp == "daml3-script")
+                $ Logger.logWarning logger
+                    "You are using an unreleased and unstable version of daml-script intended for daml3. This will break without warning."
```
Comment (remyhaemmerle-da) [inline compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs:560]: I was thinking to add the warning in the runner, but this is probably fine

**Comment by samuel-williams-da** [comment]
@remyhaemmerle-da happy to merge?

**Pull request merged**